spec/rollbar/sidekiq_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'spec_helper'
+
+unless RUBY_VERSION == '1.8.7'
+  require 'sidekiq'
+  require 'rollbar/sidekiq'
+end
+
+describe "Sidekiq integration", :reconfigure_notifier => false do
```

Comment:
Then it will blow up under 1.8.7 because that class isn't loaded in lines 3-6 above